app/components/form/fields/DateTimeRangePicker.tsx

```diff
@@ -103,7 +103,7 @@ export function DateTimeRangePicker({
   return (
     <form className="flex">
       <Listbox
-        className="z-10 w-[10rem] border-r border-r-default [&>button]:!rounded-r-none [&>button]:!border-r-0"
+        className="z-10 w-[10rem] border-r border-r-default [&_button]:!rounded-r-none [&_button]:!border-r-0"
```

Comment:
This was needed to fix the border between the `Listbox` and the button after adding a wrapping div to work around the Headless bug. We can always change it back if want after they fix it.

<img width="509" alt="image" src="https://github.com/user-attachments/assets/fff52ef5-a262-4682-96d5-355f537acc29" />